app/views/mainpages/_content.html.erb

```diff
@@ -21,14 +21,10 @@
               <p><%= result.description %></p>
             </div>
             <div class="coupon_opc">
-              <%= button_to "delete", {:controller => :cupons, :action => 'destroy', :id => result.id }, :method => :delete %>
-              <%= link_to "Edit", edit_cupon_path(result)%>
+              <%= link_to "Ver", cupon_path(result.id) %>
+              <%= ((button_to "delete", {:controller => :cupons, :action => 'destroy', :id => result.id }, :method => :delete) if current_user.id == result.user_id) if current_user %>
```

Comment:
The :method works? As long as I remember it doesn't

Reply:
The :method does work, we have tested it and everything works.